Source/InMobiAdapter.swift

```diff
@@ -78,6 +81,9 @@ final class InMobiAdapter: NSObject, PartnerAdapter {
         if status != .unknown {
             value[IMCommonConstants.IM_PARTNER_GDPR_CONSENT_AVAILABLE] = status == .granted ? String.gdprConsentAvailable : .gdprConsentUnavailable
         }
+        if let tcfString = UserDefaults.standard.string(forKey: tcfv2Key) {
+            value[IMCommonConstants.IM_GDPR_CONSENT_IAB] = tcfString
+        }
```

Comment:
InMobi doc mentions `IM_GDPR_CONSENT_AVAILABLE` and `IM_SUBJECT_TO_GDPR` as well. Do we need to provide this two values in the `value` dictionary?

https://support.inmobi.com/monetize/data-identity/unifid/unifid-sample-integration-ios/
https://support.inmobi.com/monetize/sdk-documentation/ios-guidelines/migrating-to-sdk-10xx

Reply:
Hmm. Those seem redundant since we already have `IM_PARTNER_GDPR_CONSENT_AVAILABLE` and  `IM_PARTNER_GDPR_APPLIES` but if we're only sending one or the other I don't know whether the `...PARTNER...` version is the right one.

Reply:
Yeah, I googled a bit and I don't see any documentation for these constants :(

Can you contact them for more details? https://www.inmobi.com/company/contact

Reply:
Ok, I've left a message for them via that contact form.
In the meantime, @daniel-barros you wrote this section a year ago do you happen to remember if there was a difference between `IM_PARTNER_GDPR_CONSENT_AVAILABLE` and `IM_GDPR_CONSENT_AVAILABLE`?

Reply:
I do not remember. My guess is that we copy-pasted what was being done in the old adapters.

Reply:
@CB-HaoxinLi I haven't heard back from InMobi.
Can we approve this adapter using the same labels we've been using so far, and I'll make a patch release when we have more information?

Reply:
Sure! Sorry I didn't see the email notification earlier. Please feel free to Slack me if I don't reply in a day or so. Thanks!